benchmarks/cupy/bench_fusion.py

```diff
@@ -0,0 +1,117 @@
+import functools
+import inspect
+
+from benchmarks.numpy.common import Benchmark
+from benchmarks.utils import sync
+from benchmarks.utils.helper import parameterize
+
+import cupy
+
+
+def fuse_shapes(*shape_args, **shapes_dict):
+    def decorator(func):
+        fused_func = cupy.fuse(func)
+        @functools.wraps(func)
+        def wrapper(self, fusion_param):
+            if fusion_param == 'enabled':
+                fused_func(*args)
+            elif fusion_param == 'disabled':
+                func(*args)
+            elif fusion_param == 'compile':
+                raise NotImplementedError
+                # TODO (imanishi)
+                # fused_func._memo = {}
+                # fused_func.compile(*args)
+            else:
+                raise ValueError('Invalid parameter')
+
+        uniform = cupy.random.uniform
+        args = []
+        if shape_args:
+            for name in inspect.getargspec(func).args:
+                args.append(uniform(0, 200, shape_args, cupy.float32))
+        else:
+            for name in inspect.getargspec(func).args:
+                shape = shapes_dict[name]
+                args.append(uniform(0, 200, shape, cupy.float32))
+        # TODO (imanishi)
+        # fused_func.compile(*args)
+        fused_func(*args)
+        return wrapper
+    return decorator
+
+
+@sync
+@parameterize([('fusion_param', ['enabled', 'disabled'])])
+class Fusion(Benchmark):
+    def setup(self, fusion_enabled):
+        self.x = cupy.arange(240000.0).reshape(400, 600)
```

Comment:
dtype of `x` will be `float64`. Is this expected?

Reply:
`self.x` is not referred to, so I removed it.